tests/ha/test_server_pod_failure.py

```diff
@@ -494,47 +473,29 @@ def test_writes_during_server_pod_down(self):
-        self.deployment_backup = resp[1]
-        self.deployment_name = resp[2]
+        LOGGER.info("Step 2: Shutdown random server pod safely by deleting deployment and "
+                    "verify cluster & remaining pods status")
+        resp = self.ha_obj.delete_kpod_with_shutdown_methods(
+            master_node_obj=self.node_master_list[0], health_obj=self.hlth_master_list[0],
+            pod_prefix=[const.SERVER_POD_NAME_PREFIX], down_method=const.RESTORE_DEPLOYMENT_K8S,
+            event=event)
+        # Assert if empty dictionary
+        assert_utils.assert_true(resp[1], "Failed to shutdown/delete pod")
+        pod_name = list(resp[1].keys())[0]
+        self.deployment_name = resp[1][pod_name]['deployment_name']
+        self.deployment_backup = resp[1][pod_name]['deployment_backup']
         self.restore_pod = self.deploy = True
-        self.restore_method = const.RESTORE_DEPLOYMENT_K8S
-        LOGGER.info(
-            "Step 2: Successfully shutdown/deleted server pod %s by deleting deployment.", pod_name)
-
-        LOGGER.info("Step 3: Check cluster status")
-        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
-        assert_utils.assert_false(resp[0], resp)
-        LOGGER.info(
-            "Step 3: Cluster has some failures due to server pod %s has gone down.", pod_name)
-
-        LOGGER.info("Step 4: Check services status that were running on server pod %s", pod_name)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=[pod_name], fail=True,
-                                                           hostname=hostname)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Services of server pod %s are in offline state", pod_name)
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown server pod %s safely. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
 
-        pod_list.remove(pod_name)
-        LOGGER.info("Step 5: Check services status on remaining pods %s", pod_list)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=pod_list, fail=False)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services status on remaining pod are in online state")
-        event.clear()
         thread.join()
+        LOGGER.info("Thread has joined.")
         responses = {}
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
-        LOGGER.info("Step 6: Verify status for In-flight WRITEs while server pod %s is going down "
+        LOGGER.info("Step 3: Verify status for In-flight WRITEs while server pod %s is going down "
                     "should be failed/error.", pod_name)
```

Comment:
instead of should be, is*

Reply:
> instead of should be, is*

This is before verification statement, where we can should. In concluding its is* only.